fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -261,10 +263,22 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management)
     }
 
     private fun openAMO() {
+        openLinkInNewTab(AMO_HOMEPAGE_FOR_ANDROID)
+    }
+
+    private fun openLearnMoreLink(link: AddonsManagerAdapterDelegate.LearnMoreLinks, addon: Addon) {
+        val url = when (link) {
+            AddonsManagerAdapterDelegate.LearnMoreLinks.BLOCKLISTED_ADDON ->
+                "${BuildConfig.AMO_BASE_URL}/android/blocked-addon/${addon.id}/"
+        }
+        openLinkInNewTab(url)
+    }
```

Comment:
It looks like we don't have a SUMO article for the blocklist page, for SUMO articles we have a lot of infrastructure that [handles language and FX versioning](https://github.com/mozilla-mobile/firefox-android/blob/119c4b22570ff7bde4f0b1cf54dc7ed3bb4a74d2/fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt#L42-L79). Versioning/translation is already handle by the AMO website?

Reply:
> It looks like we don't have a SUMO article for the blocklist page, for SUMO articles we have a lot of infrastructure that [handles language and FX versioning](https://github.com/mozilla-mobile/firefox-android/blob/119c4b22570ff7bde4f0b1cf54dc7ed3bb4a74d2/fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt#L42-L79). Versioning/translation is already handle by the AMO website?

Oh, that's what I was looking for if we ever need to link to an in-product SUMO page, thanks! Yes, AMO has translation in place and - for blocklisted add-ons - we should point users to AMO, not SUMO.